app/controllers/api/enterprises_controller.rb

```diff
@@ -17,7 +17,8 @@ def create
 
       @enterprise = Enterprise.new(params[:enterprise])
       if @enterprise.save
-        render text: @enterprise.id, :status => 201
+        render text: Api::Admin::BasicEnterpriseSerializer.new(@enterprise).to_json,
```

Comment:
From memory, this is something to do with `Spree::Api::BaseController` interpreting `render :json` as "render a rabl template". I'm not sure I ever got to the bottom of it...